doc/classes/EditorInterface.xml

```diff
@@ -79,6 +91,7 @@
 			<argument index="1" name="for_property" type="String" default="&quot;&quot;">
 			</argument>
 			<description>
+				Inspects the given [code]object[/code] for the given property.
```

Comment:
I don't understand the purpose of this method. It doesn't return anything - what does it actually do?

Reply:
It selects the given object in the Editor's inspector and scrolls to the given property, if it exists. The programmatic equivalent of clicking on a node in the scene hierarchy.